### PR TITLE
feature/FOUR-14755: The new launch screen is not updated immediately

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
The new launch screen is not updated immediately

## Solution
After configure the screen the ProcessInfo need to refresh in order to show the Screen

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-14755

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-6171336eb4.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-6171336eb4.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-6171336eb4.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/ProcessesCatalogue.vue [hunk 1]
```diff
@@ -187,6 +187,8 @@ export default {
      */
     checkSelectedProcess() {
       if (this.process) {
+        this.process.launchpad = this.launchpad;
+        console.log(this.process, this.launchpad);
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessesCatalogue.vue:191]: remove this

**Review by fagubla — CHANGES_REQUESTED** [review]

**Pull request closed without merging**